inc/Engine/Admin/Settings/Page.php

```diff
@@ -679,30 +679,6 @@ private function assets_section() {
 						'button_label' => __( 'Activate minify CSS', 'rocket' ),
 					],
 				],
-				'minify_concatenate_css'       => [
```

Comment:
We need to move the `minify_concatenate_css` value to the hidden fields array to preserve the value in case of revert